.github/workflows/env-setup-deploy-redis.yml

```diff
@@ -0,0 +1,36 @@
+name: Env Setup - Deploy Redis in Openshift
+run-name: Env Setup - Deploy Redis in Openshift from ${{ github.ref_name }} to ${{ inputs.environment }}
+
+concurrency: redis-setup
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment"
+        required: true
+        type: environment
+
+jobs:
+  deployRedis:
+    name: Deploy Redis in Openshift
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment }}
+    env:
+      NAMESPACE: ${{ secrets.OPENSHIFT_ENV_NAMESPACE }}
+    steps:
+      - name: Checkout Target Branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref_name }}
+      - name: Log in to OpenShift
+        run: |
+          oc login --token=${{ secrets.SA_TOKEN }} --server=${{ vars.OPENSHIFT_CLUSTER_URL }}
+      - name: Init Redis
+        working-directory: "./devops/"
+        run: |
+          make init-redis
+      - name: Deploy Redis
+        working-directory: "./devops/"
+        run: |
+          make deploy-redis
```

Comment:
can we have a needs here to ensure it runs depending on init-redis being complete. ??